From 42d61ee8d26ba09096a5acb759f6ec3df5912524 Mon Sep 17 00:00:00 2001
From: Felix Yan <felixonmars@archlinux.org>
Date: Wed, 14 Sep 2022 23:44:33 +0300
Subject: [PATCH] Remove unused setuptools dependency

`setuptools` or `pkg_resources` are not runtime dependency for `profilestats`, let's remove it.
---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.py b/setup.py
index b7be6f1..e13e5b7 100755
--- a/setup.py
+++ b/setup.py
@@ -34,7 +34,6 @@
     ],
     license='BSD',
     install_requires=[
-        'setuptools',
         'pyprof2calltree',
     ],
     py_modules=['profilestats'],
